pepper-apis/src/test/java/org/broadinstitute/ddp/RateLimitTest.java

```diff
@@ -0,0 +1,127 @@
+package org.broadinstitute.ddp;
+
+import static org.broadinstitute.ddp.route.IntegrationTestSuite.startupTestServer;
+import static org.broadinstitute.ddp.route.IntegrationTestSuite.tearDown;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.Callable;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.atomic.AtomicInteger;
+
+import com.typesafe.config.Config;
+import org.apache.http.HttpResponse;
+import org.apache.http.client.fluent.Request;
+import org.broadinstitute.ddp.constants.ConfigFile;
+import org.broadinstitute.ddp.constants.RouteConstants;
+import org.broadinstitute.ddp.route.IntegrationTestSuite;
+import org.broadinstitute.ddp.route.RouteTestUtil;
+import org.broadinstitute.ddp.util.ConfigManager;
+import org.eclipse.jetty.http.HttpStatus;
+import org.junit.AfterClass;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class RateLimitTest extends IntegrationTestSuite.TestCase {
```

Comment:
@MocanaAtBroad here's the test!  Leveraged a lot of config overrride stuff from the password rotation odyssey.